### PR TITLE
📖 update succeeded to Complete

### PR DESCRIPTION
**Pull request by daniel-hutao**
📖 : JobConditionType is "Complete" or "Failed", so I update succeeded to Complete in this file.

-> k8s.io/api@v0.17.2/batch/v1/types.go:165

### CODE REVIEW

File: docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go [hunk 1]
```diff
@@ -144,7 +144,7 @@ func (r *CronJobReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 		root object.  Instead, you should reconstruct it every run.  That's what we'll
 		do here.
 
-		We can check if a job is "finished" and whether it succeeded or failed using status
+		We can check if a job is "finished" and whether it Complete or Failed using status
```
Comment (gabbifish) [inline docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go:147]: small grammatical nit:
```suggestion
		We can check if a job is finished and whether it has state "Complete" or "Failed" using status
```

File: docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go [hunk 2]
```diff
@@ -155,7 +155,7 @@ func (r *CronJobReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	var mostRecentTime *time.Time // find the last run so we can update the status
 
 	/*
-		We consider a job "finished" if it has a "succeeded" or "failed" condition marked as true.
+		We consider a job "finished" if it has a "Complete" or "Failed" condition marked as true.
```
Comment (gabbifish) [inline docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go:158]:
```suggestion
		We consider a job finished if it has a "Complete" or "Failed" condition marked as true.
```

**Review by gabbifish — COMMENTED** [review]
Included some tiny suggestions to improve grammar. Otherwise LGTM! :)

**Comment by camilamacedo86** [comment]
It shows great 👍 Could you please address the @gabbifish suggestions and update the title with the mojo: https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md#pr-process?

File: docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go (hunk 1 above)
Comment (daniel-hutao) [reply docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go:147]: right

**Comment by daniel-hutao** [comment]
> It shows great 👍 Could you please address the @gabbifish suggestions and update the title with the mojo: https://github.com/kubernetes-sigs/kubebuilder/blob/master/CONTRIBUTING.md#pr-process? Also, please ensure that your commits are squashed.

I've added the 📖 to my commit's title. ^_^

**Comment by gabbifish** [comment]
@daniel-hutao once you get a chance to incorporate the suggestions above, I'm ready to give this PR an lgtm! :)

**Comment by daniel-hutao** [comment]
@gabbifish Isn't that what I should do?
add 📖 to my title：
📖 update succeeded to Complete #1677

**Comment by gabbifish** [comment]
@daniel-hutao please see and merge the github suggestions in my review above, then this should be ready for merge.

File: docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go (hunk 2 above)
Comment (camilamacedo86) [reply docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go:158]: @daniel-hutao shows that it is pushing the nit changes.

File: docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go (hunk 1 above)
Comment (camilamacedo86) [reply docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go:147]: @daniel-hutao shows that it is pushing the nit changes.

**Comment by daniel-hutao** [comment]
> @daniel-hutao please see and merge the github suggestions in my review above, then this should be ready for merge.

@gabbifish I'm sorry, this's the first time I use github suggestions function. I didn't understand it before. I will know it next time. ^_^
Am I right now?

**Comment by camilamacedo86** [comment]
Hi daniel-hutao, 

Could you please squash your commits?

**Comment by daniel-hutao** [comment]
> Hi daniel-hutao,
> 
> Could you please squash your commits?

@camilamacedo86 Excuse me, should I rebase the 2 commit in my repo and then create a new pull request to this repo? Because I don't found the place to finish squash here, and it's the first time I deal with this scene, sorry!

**Comment by daniel-hutao** [comment]
> Hi daniel-hutao,
> 
> Could you please squash your commits?

It seems to be solved @camilamacedo86

**Review by gabbifish — APPROVED** [review]
/lgtm

**Pull request merged**